projects/jsoncpp/jsoncpp_fuzz_proto.cc

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+#include "json.pb.h"
+#include "json_proto_converter.h"
+#include "libprotobuf-mutator/src/libfuzzer/libfuzzer_macro.h"
+
+#include <cstdint>
+#include <json/config.h>
+#include <json/json.h>
+#include <memory>
+#include <string>
+#include <iostream>
+#include <cstddef>
+#include <stdint.h>
+#include <iostream>
+
+namespace Json {
+class Exception;
+}
+
+extern "C" int FuzzJson(const char* data_str, size_t size, int32_t hash_settings) {
```

Comment:
we don't need `extern "C"` as this function isn't being exported, also can change return type to `void` and delete lines 57-58